tutorials/introductory/usage.py

```diff
@@ -290,44 +290,60 @@ def my_plotter(ax, data1, data2, param_dict):
 # "interactive backends") and hardcopy backends to make image files
 # (PNG, SVG, PDF, PS; also referred to as "non-interactive backends").
 #
-# There are three ways to configure your backend. If they conflict each other,
-# the method mentioned last in the following list will be used, e.g. calling
-# :func:`~matplotlib.use()` will override the setting in your ``matplotlibrc``.
+# Selecting a backend
+# -------------------
 #
-# #. The :rc:`backend` parameter in your ``matplotlibrc`` file (see
-#    :doc:`/tutorials/introductory/customizing`)::
+# There are three ways to configure your backend:
+#
+# 1. The :rc:`backend` parameter in your ``matplotlibrc`` file
+# 2. The :envvar:`MPLBACKEND` environment variable
+# 3. The function :func:`matplotlib.use`
+#
+# A more detailed description is given below.
+#
+# If multiple of these are configurations are present, the last one from the
+# list takes precedence; e.g. calling :func:`matplotlib.use()` will override
+# the setting in your ``matplotlibrc``.
+#
+# If no backend is explicitly set, Matplotlib automatically detects a usable
+# backend based on what is available on your system and on whether a GUI event
+# loop is already running. On Linux, if the environment variable
+# :envvar:`DISPLAY` is unset, the "event loop" is identified as "headless",
+# which causes a fallback to a noninteractive backend (agg).
+#
+# Here is a detailed description of the configuration methods:
+#
+# #. Setting :rc:`backend` in your ``matplotlibrc`` file::
 #
 #        backend : qt5agg   # use pyqt5 with antigrain (agg) rendering
 #
-#    If no backend is explicitly set in the ``matplotlibrc`` file, Matplotlib
-#    automatically detects a usable backend based on what is available on your
-#    system and on whether a GUI event loop is already running.
+#    See also :doc:`/tutorials/introductory/customizing`.
+#
+# #. Setting the :envvar:`MPLBACKEND` environment variable:
 #
-#    On Linux, if the environment variable :envvar:`DISPLAY` is unset, the
-#    "event loop" is identified as "headless", which causes a fallback to a
-#    noninteractive backend (agg).
+#    You can set the environment variable either for your current shell or for
+#    a single script.
 #
-# #. Setting the :envvar:`MPLBACKEND` environment variable, either for your
-#    current shell or for a single script.  On Unix::
+#    On Unix::
 #
-#         > export MPLBACKEND=module://my_backend
+#         > export MPLBACKEND=qt5agg
 #         > python simple_plot.py
 #
-#         > MPLBACKEND="module://my_backend" python simple_plot.py
+#         > MPLBACKEND="qt5agg" python simple_plot.py
```

Comment:
quotes not needed (either there or when using module://) (note that on Windows you don't use quotes either, and if anything one should prefer single to double quotes in the shell anyways...)